app/assets/stylesheets/base.css.scss

```diff
@@ -81,9 +81,17 @@ a {
     margin: 0 auto;
 }
 
+.profpiccircle {
```

Comment:
For multi-word classes it's generally good practice to separate the words with a dash `-`, e.g. `.prof-pic-circle`. Also I would expand this to be a little clearer, e.g. `profile-pic-circle` (I first read this instinctively as 'professor-pic-circle'...).

For dashes that goes for e.g. `dash-profile` and `total-points` too (below).